scripts/docker-setup.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+RED='\033[1;31m'
+NC='\033[0m' # No Color
+FORK=automaticrippingmachine
+TAG=latest
+function usage() {
+    echo -e "\nUsage: docker_setup.sh [OPTIONS]"
+    echo -e " -f <fork>\tSpecify the fork to pull from on DockerHub. \n\t\tDefault is \"$FORK\""
+    echo -e " -t <tag>\tSpecify the tag to pull from on DockerHub. \n\t\tDefault is \"$TAG\""
+}
+
+while getopts 'f:t:' OPTION
+do
+    case $OPTION in
+    f)    FORK=$OPTARG
+          ;;
+    t)    TAG=$OPTARG
+          ;;
+    ?)    usage
+          exit 2
+          ;;
+    esac
+done
+IMAGE="$FORK/automatic-ripping-machine:$TAG"
+
+function install_reqs() {
+    apt update -y && apt upgrade -y
+    apt install -y curl lsscsi
+}
+
+function add_arm_user() {
+    echo -e "${RED}Adding arm user${NC}"
+    # create arm group if it doesn't already exist
+    if ! [[ "$(getent group arm)" ]]; then
+        groupadd arm
+    else
+        echo -e "${RED}arm group already exists, skipping...${NC}"
+    fi
+
+    # create arm user if it doesn't already exist
+    if ! id arm >/dev/null 2>&1; then
+        useradd -m arm -g arm
+        passwd arm
+    else
+        echo -e "${RED}arm user already exists, skipping...${NC}"
+    fi
+    usermod -aG cdrom,video arm
+}
+
+function launch_setup() {
+    # install docker
+    if [ -e /usr/bin/docker ]; then
+        echo -e "${RED}Docker installation detected, skipping...${NC}"
+    else
+        echo -e "${RED}Installing Docker${NC}"
+        # the convenience script auto-detects OS and handles install accordingly
+        curl -sSL https://get.docker.com | bash
+        usermod -aG docker arm
+    fi
+}
+
+function pull_image() {
+    echo -e "${RED}Pulling image from $IMAGE${NC}"
+    sudo -u arm docker pull "$IMAGE"
+}
+
+function setup_mountpoints() {
+    echo -e "${RED}Creating mount points${NC}"
+    for dev in /dev/sr?; do
+        sudo mkdir -p "/mnt$dev"
+    done
+    sudo chown arm:arm /mnt/dev/sr*
+}
+
+function save_start_command() {
+    url="https://raw.githubusercontent.com/automatic-ripping-machine/automatic-ripping-machine/v2_devel/scripts/docker/start_arm_container.sh"
```

Comment:
the `v2_devel` branch depreciated and is no longer writeable. It should link to main instead.

Reply:
Ah, fixed that in the test environment but never committed

Reply:
Ok, just ping when its ready for re-review.